src/Microsoft.AspNet.Mvc.ModelBinding/Validation/DataTypeAttributeAdapter.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
     /// A validation adapter that is used to map <see cref="DataTypeAttribute"/>'s to a single client side validation
     /// rule.
     /// </summary>
-    public class DataTypeAttributeAdapter : DataAnnotationsModelValidator
+    public class DataTypeAttributeAdapter : DataAnnotationsClientModelValidator
```

Comment:
Just a thought - should the now client-only validator adapters move into a separate namespace? Like `.ClientValidators` or `.Validators.Client`?